projects/components/src/modal/modal-container.component.ts

```diff
@@ -85,6 +87,24 @@ export class ModalContainerComponent {
     this.visible = false;
     this.popoverRef.close();
   }
+
+  private isModalDimension(size: unknown): size is ModalDimension {
+    // tslint:disable-next-line:strict-type-predicates
+    return typeof size === 'object' && size !== null && (size as ModalDimension).width !== undefined;
```

Comment:
you should probably use 'in' operator here

Reply:
done